### PR TITLE
[FrameworkBundle] Don't log twice with the error handler

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #16980 |
| License | MIT |
| Doc PR | - |

ping @Tobion @yapro does it work as expected with this patch?

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -154,7 +154,7 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setDefinition('debug.event_dispatcher.parent', $definition);
             $container->setAlias('event_dispatcher', 'debug.event_dispatcher');
         } else {
-            $definition->replaceArgument(2, E_COMPILE_ERROR | E_PARSE | E_ERROR | E_CORE_ERROR);
+            $definition->replaceArgument(1, null);
```
Comment (dupuchba) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: it's supposed to throw the `OutOfBoundsException`right ?

Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: I'm nit sure to understand what you mean here

Comment (dupuchba) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: Sorry, didn't check well the `replaceArgument`function :-)

**Comment by yapro** [comment]
@nicolas-grekas it does not work because kernel.debug is true

**Comment by nicolas-grekas** [comment]
@yapro I also changed the `kernel.debug is true` code path...

**Comment by yapro** [comment]
@nicolas-grekas https://github.com/symfony/symfony/pull/17202/files#diff-0e793081ceb720201745c982a568903fR145

**Comment by yapro** [comment]
@nicolas-grekas now it work, thank you

**Comment by nicolas-grekas** [comment]
I just checkouted this on a 2.8 std-edition, works well in prod and dev
Without further comments, I'm going to merge this soon.
ping @Tobion

**Comment by yapro** [comment]
@nicolas-grekas thx you, but why not 2.7

**Comment by nicolas-grekas** [comment]
I didn't check, does the issue exist on 2.7?

**Comment by yapro** [comment]
@nicolas-grekas yes

**Comment by nicolas-grekas** [comment]
@yapro would you like to open the PR on 2.7?

**Comment by yapro** [comment]
@nicolas-grekas yes, tnx you

**Comment by yapro** [comment]
@nicolas-grekas may be you forget?

**Comment by xabbuh** [comment]
@yapro I guess @nicolas-grekas meant if you wanted to create the PR. :) But never mind I opened #17433 to backport this to Symfony 2.7.

**Comment by yapro** [comment]
@xabbuh thx you :)

**Pull request merged**